runtime/src/bucket_map_holder.rs

```diff
@@ -366,9 +384,8 @@ pub mod tests {
     #[test]
     fn test_throttle() {
         solana_logger::setup();
-        let bins = 100;
+        let bins = 128;
         let test = BucketMapHolder::<u64>::new(bins, &Some(AccountsIndexConfig::default()), 1);
-        assert!(!test.is_disk_index_enabled());
```

Comment:
should we keep this assert, but flip the boolean, to check that by default disk_index is enabled?

Reply:
this test should be completely independent of disk index or not. I'm not sure why it was there. Very possibly left over from copying test code. I almost certainly wrote this test.